dashboards-observability/public/components/common/live_tail/live_tail_button.tsx

```diff
@@ -5,54 +5,51 @@
 
 //Define pop over interval options for live tail button in your plugin
 
-import { EuiButton } from '@elastic/eui';
-import React, { useMemo } from 'react';
-import { LiveTailProps } from 'common/types/explorer';
+import { EuiButton } from "@elastic/eui";
+import React, { useMemo } from "react";
+import { LiveTailProps } from "common/types/explorer";
 
 //Live Tail Button
 export const LiveTailButton = ({
-    isLiveTailOn,
-    isLiveTailPopoverOpen,
-    setIsLiveTailPopoverOpen,
-    liveTailName,
-    dataTestSubj,
+	isLiveTailOn,
```

Comment:
are you not using prettier as formatter? i don't think we use tabs

Reply:
i did format it but unfortunately it wasn't using prettier. now changed it to prettier